src/dates/format-date.js

```diff
@@ -123,7 +123,7 @@ formatters.S = function(date, formatLength) {
     const milliseconds = date.getMilliseconds();
     let result;
     if (milliseconds !== 0) {
-        result = String(date.getMilliseconds() / 1000).split(".")[1].substr(0, formatLength);
+        result = String(date.getMilliseconds() / 1000).split(".")[1].substr(0, formatLength).padEnd(formatLength, '0');
```

Comment:
Do we have a polyfill for this? IE doesn't support `padEnd` natively.
https://caniuse.com/pad-start-end

Reply:
Turns out we had a ready `pad` util, used 2 lines below, and also everywhere in this file. 😆